src/skin/imgloader.cpp

```diff
@@ -25,6 +26,9 @@ QImage* ImgLoader::getImage(const QString& fileName, double scaleFactor) const {
         }
     }
     if (scaleFactor > 1.0) {
+#else
+    if (qApp->devicePixelRatio() > 1.0) {
```

Comment:
Additionally, I don't feel good with spamming all the code structure with a parameter. Qt5 is designed to make application capable of HiDPI with minimal modifications.